modules/wrapper/rc/options.nix

```diff
@@ -3,13 +3,19 @@
   lib,
   ...
 }: let
+  inherit (lib.modules) mkRemovedOptionModule;
   inherit (lib.options) mkOption mkEnableOption literalMD literalExpression;
   inherit (lib.strings) optionalString;
-  inherit (lib.types) str oneOf attrs lines listOf either path bool;
+  inherit (lib.types) str attrs lines listOf either path bool;
   inherit (lib.nvim.types) dagOf;
   inherit (lib.nvim.lua) listToLuaTable;
+
   cfg = config.vim;
 in {
+  imports = [
+    (mkRemovedOptionModule ["vim" "configRC"] "Migrate your configRC sections to lua, and add them to luaConfigRC instead.")
```

Comment:
It would be nice to add
1. A "migration guide" to the top of the changelog (like we did for setupOpts in 0.6)
2. Mention said guide here